### ISSUE TITLE
Questions about the Crop Size

### ISSUE THREAD

**Issue by qianyuzqy** [vikolss/DACS#2]
Hi, thank you so much for your open-source code! It is well-organized and great!

I found that the crop size is (128, 512) in your config.json, while you mentioned that the images are randomly cropped to (512, 512) in your paper?  I was wondering what is the final crop size you used to get the best performance of 52.14 in the final experiment?

Another question is that is it necessary to resize the cityscapes image from (h,w) to (h/2, h/w)  and to resize the gtav image from (760,1280) to (300, 600) during the process of the random crop? It is not mentioned in your paper, will it affect the performance a lot?

Thank you very much!

All the best,

**Comment by WilhelmT** [comment]
Hi, thank you for your interest

You are right, the crop size used in the paper was indeed 512 and has been changed accordingly.

The Cityscapes images are resized to 512x1024 before cropping, and the gta images are resized to 760x1280 (same as synthia). This was consistent with existing work using DeepLabv2 for these benchmarks. We did not experiment with how different choices would affect performance.
